### DISCUSSION TITLE
Product images as URL

### DISCUSSION THREAD

**Discussion by jesiiu** [PrestaShop/PrestaShop#36797] [Q&A]
Hello, can someone tell me if I can somehow assign images to a product as a URL without uploading it directly to the server as a file? There will be about 8000 products in my store and after uploading all the images the size of the store on disk is more than 30GB. Terrible waste of space and on top of all that it slows down the performance....

**Comment by kpodemski** [comment]
> Terrible waste of space and on top of all that it slows down the performance....

What would slow down the shop in that case? I imagine that loading images from a third-party server without correct caching could be an issue, but having them on the same server and using the CDN option should be fine. Also, you can use Cloudflare to speed things up by caching static assets like images.

If you want, search for PrestaShop S3 Images or PrestaShop CDN Images modules that allow you to set up external servers to host images.

**Comment by jesiiu** [comment]
Niby tak ale z drugiej strony zarządzanie zdjęciami które są wygrywane bezpośrednio na sklep przez integracje webservice jest tragiczne, przy każdej aktualizacji produktu muszę usuwać powiązane zdjęcia do produtku a następnie pobierać cały pakiet nowych i wgrywać od nowa. W momencie w którym odbywałoby się to przez CDN albo po prostu URL to nie byłoby żadnego problemu, kwestia sprawdzenia czy produkt ma wszystkie URL'e przypisane
